sdk/eventhub/Azure.Messaging.EventHubs.Processor/tests/EventProcessorClient/EventProcessorClientTests.cs

```diff
@@ -666,24 +972,38 @@ private static async IAsyncEnumerable<PartitionEvent> MockPartitionEventEnumerab
         ///   Serves as a non-functional connection for testing consumer functionality.
         /// </summary>
         ///
-        private class MockConnection : EventHubConnection
+        internal class MockConnection : EventHubConnection
         {
             public MockConnection(string namespaceName = "fakeNamespace",
-                                  string eventHubName = "fakeEventHub") : base(namespaceName, eventHubName, CreateCredentials())
-            {
-            }
+                                  string eventHubName = "fakeEventHub",
+                                  EventHubConnectionOptions options = null) : base(namespaceName, eventHubName, CreateCredentials(), options)
+            { }
+
 
             private static EventHubTokenCredential CreateCredentials()
             {
                 return new Mock<EventHubTokenCredential>(Mock.Of<TokenCredential>(), "{namespace}.servicebus.windows.net").Object;
             }
         }
 
+        /// <summary>
+        ///   Serves a mock MockEventHubProperties.
```

Comment:
nit: maybe it would sound better if we changed it to `Serves as a mock <see cref="EventHubProperties" />.`?